examples/tutorial.py

```diff
@@ -60,30 +64,36 @@
 print_open_orders(orders)
 
 # Place a limit order 20% below the best bid
-best_bid = float(book.bids[0].price)
-limit_price = round(best_bid * 0.8)
-quantity = 1.0
+best_bid = book.bids[0].price
+limit_price = best_bid * Decimal(0.8)
+quantity = Decimal(1)
 account = accounts[0]
 print()
 order = None
+
+assert isinstance(market.kind, MarketFieldsKindExchangeMarketKind)
```

Comment:
I know this is the tutorial, but this is so that there are no type errors.

I kind of hate that we have to do this to not get type errors, what are your thoughts?
should we ignore type errors or just type this